astro.config.mjs

```diff
@@ -1,4 +1,5 @@
 import { defineConfig } from 'astro/config';
+import flexsearchPlugin from './src/plugins/flexsearch';
```

Comment:
I thought this doesn't work, is this why the build is broken?

Reply:
It wasn't because of this. I've fixed the build.